### PR TITLE
Add support for making an ingress private in spec

### PR DESCRIPTION
**Pull request by tjhiggins**

### CODE REVIEW

File: src/commands/dev/index.ts [hunk 1]
```diff
@@ -553,12 +359,70 @@ export default class Dev extends BaseCommand {
...
+    process.on('SIGINT', () => {
+      is_exiting = true;
+    });
+
+    compose_process.on('exit', () => {
+      if (!flags.detached) {
+        fs.removeSync(compose_file);
+      }
+      // eslint-disable-next-line no-process-exit
+      process.exit(0);
```
Comment (tjhiggins) [inline src/commands/dev/index.ts:389]: Possible I should be using `this.exit`

File: src/commands/dev/index.ts [hunk 2]
```diff
@@ -367,6 +169,11 @@ export default class Dev extends BaseCommand {
       sensitive: false,
       default: false,
     }),
+    'wait-timeout': Flags.string({
+      description: '[default: 10m] Time to wait for services to be ready/healthy before detaching.',
```
Comment (TylerAldrich) [inline src/commands/dev/index.ts:173]: It looks like oclif adds the default text automatically, it shows up twice:

image

**Review by TylerAldrich — CHANGES_REQUESTED** [review]
Code changes all LGTM, did a bunch of testing with dev and interrupting dev with failing/restarting containers and everything seems to be working as expected. Just one tiny --help text issue

File: src/commands/dev/index.ts (hunk 2 above)
Comment (tjhiggins) [reply src/commands/dev/index.ts]: Good find. Found a couple other places where it was duplicated and fixed.

**Review by TylerAldrich — APPROVED** [review]

**Pull request merged**